src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import "./1-helpers/variables";
+@import "./1-helpers/mixins";
+@import "./1-helpers/reset";
+@import "./1-helpers/common";
+
+@import "./2-components/card";
+@import "./2-components/stars";
+@import "./3-UI/card-button";
```

Comment:
It's not so important in this case, but why do you name folders with such a pattern as "1-helpers"?

Reply:
I thought about how to structure my project into folders and looked at the real world example there I saw a similar structure and I liked it